src/main/kotlin/org/jitsi/nlj/rtp/TransportCcEngine.kt

```diff
@@ -230,8 +230,7 @@ class TransportCcEngine(
         MAX_OUTGOING_PACKETS_HISTORY,
         /* We don't want to clone [PacketDetail] objects that get put in the tracker. */
         { it },
-        true,
-        clock
+        clock = clock
```

Comment:
My main issue here was with the bare `true`, but if that's no longer needed i think it's safe to omit the named arg for `clock`, as that's pretty self-explanatory.

Reply:
The named `clock` is necessary to avoid the bare `true`; true is the default value for the third argument (`synchronized`), and we can only get its default if we use a named argument for the fourth argument.

Reply:
Ah, gotcha.